### PR TITLE
* Semi implement #728, with points 2 & 4 outstanding

### PR DESCRIPTION
**Pull request by Wagnerp**
* Semi implement #728, with points 2 & 4 outstanding

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs [hunk 1]
```diff
@@ -251,6 +251,91 @@ private void UpdateIcon()
 
         private void UpdateButtons()
         {
+#if NET6_0
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs:254]: This scope is "Way too big"
Just needed around the single case line statement, 
With `#else` binging the actual int number

File: Source/Krypton Components/Krypton.Toolkit/General/GlobalStrings.cs [hunk 2]
```diff
@@ -62,6 +66,20 @@ public GlobalStrings()
         /// </summary>
         /// <returns>True if all values are defaulted; otherwise false.</returns>
         [Browsable(false)]
+#if NET6_0
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/General/GlobalStrings.cs:69]: Remove the `#if`
Remove the `#else`

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs:254]: Yes, just thought about that... will "fix" now

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs]: Fixed

File: Source/Krypton Components/Krypton.Toolkit/General/GlobalStrings.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/GlobalStrings.cs]: Fixed

**Comment by Wagnerp** [comment]
Resolved conflicts

**Comment by Smurf-IV** [comment]
> Resolved conflicts

Still need to resolve the last 2 Review comments.

**Comment by Wagnerp** [comment]
> > Resolved conflicts
> 
> Still need to resolve the last 2 Review comments. (And the conflict)

Done :)

**Pull request merged**